ethicalads-theme/templates/ea/ad-design-and-specs.html

```diff
@@ -0,0 +1,226 @@
+{% extends "page.html" %}
+
+
+{% block content %}
+<section class="py-10">
+  <div class="container">
+
+    <div class="mb-8">
+      <h1 class="font-weight-bold text-center">EthicalAds ad creative specs &amp; design</h1>
+      {% if page.subtitle %}<p class="lead text-center">{{ page.subtitle }}</p>{% endif %}
+    </div>
+
+    <div class="row row-cols-1 row-cols-md-3 mb-8">
+
+      <!-- The body -->
+      <div class="col mb-2">
+        <div class="card shadow-sm h-100">
+          <div class="card-body">
+            <h3 class="card-title">Ad body</h3>
+            <p class="card-text">The ad body can be <strong>up to 100 characters</strong> and can include an optional bold headline and call to action.</p>
+
+            <div class="text-center bg-light small rounded py-3">
+              <strong>Read the Docs for Business:</strong>
+              <span> You write the docs. We do the rest. </span>
+              <strong>Try it today!</strong>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- The image -->
+      <div class="col mb-2">
+        <div class="card shadow-sm h-100">
+          <div class="card-body">
+            <h3 class="card-title">Ad image</h3>
+            <p class="card-text">The image is <strong>240*180 pixels</strong> and is displayed at 120*90px on high resolution displays. The image can be optional to support publishers who use text-only ads.</p>
```

Comment:
This feels confusing. Isn't it displayed at 240x180 on high res displays?
```suggestion
            <p class="card-text">The image is <strong>240*180 pixels</strong> to support high resolution displays, and is displayed at 120*90px. The image can be optional to support publishers who use text-only ads.</p>
```

Reply:
It is displayed at 120*90 on all devices. We have double the resolution to it looks sharp on phones.